### PR TITLE
Edit contact feature by dikshya

### PR DESCRIPTION
**Pull request by Dikshyapoudel12**
#Description:
This is a feature file for the Edit the contact
##Issue:
https://github.com/SagarGi/FeatureFiles/issues/6

### CODE REVIEW

File: Features4/tests/feature/editContact.feature [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+       
+
+Scenario:Edit specific contact information
+     Given the following contacts have been created 
+        |name     |email               |phone       |contact-type  |
+        |deekshya |deekshyap@gmail.com |9567989999  | personal     |
+        |laxmi    |laxmi@gmail.com     |9689999978  | personal     | 
+        |dipesh   |dipesh@gmail.com    |4678990000  | professional |
+        |shyam    |shyam@gmail.com     |6788999999  | professional |
+
```
Comment (SagarGi) [inline Features4/tests/feature/editContact.feature]:
```suggestion
```

File: Features4/tests/feature/editContact.feature [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    When the user makes the following updates with email "deekshyap@gmail.com" :
```
Comment (SagarGi) [inline Features4/tests/feature/editContact.feature]:
```suggestion
    When the user makes the following updates with email "deekshyap@gmail.com":
```

File: Features4/tests/feature/editContact.feature [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        | Name     | Email                 | Phone | Contact Type    |
+        | deekshya | deekshyppp@gmail.com  |       | professional    |
+    Then the contact with the email "deekshyap@gmail.com" should be updated
```
Comment (SagarGi) [inline Features4/tests/feature/editContact.feature]:
```suggestion
    Then the contact with the email "deekshyap@gmail.com" should be updated as:
```

File: Features4/tests/feature/editContact.feature [hunk 4]
```diff
@@ -0,0 +1,37 @@
+Feature:
+    As a user
+    I want to edit contact
+    So that I can correct mistakes 
+
+Background:
+    Given the user has logged in with the valid email "deekshyap@gmail.com" and password "123456"
+    And  navigated to Contact Fox page
```
Comment (kiranadh1452) [inline Features4/tests/feature/editContact.feature]:
```suggestion
    And the user has navigated to Contact Fox page
```

Comment (kiranadh1452) [inline Features4/tests/feature/editContact.feature]:
```suggestion
    And the user has navigated to Contact Fox page
```

File: Features4/tests/feature/editContact.feature [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        |name     |email               |phone       |contact-type  |
+        |deekshya |deekshyap@gmail.com |9567989999  | personal     |
+    
+    When the user makes the following updates with email "deekshyap@gmail.com" :
+        | Name     | Email                 | Phone     | Contact Type    |
+        | deekshya | deekshyppp@gmail.com  | 956798999 | personal    |
+    Then the updated contact with email "deekshyap@gmail.com" in the contact list should be:
+        |name    | email               |phone       |contactType |
+        |deekshya|deekshyppp@gmail.com |956798999 |personal    |
+
```
Comment (kiranadh1452) [inline Features4/tests/feature/editContact.feature]: Fix the identation.

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

File: Features4/tests/feature/editContact.feature [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+        | deekshya | deekshyppp@gmail.com  |       | professional    |
+    Then the contact with the email "deekshyap@gmail.com" should be updated
+        | Name     | Email                 | Phone      | Contact Type |
+        | deekshya | deekshyppp@gmail.com  | 956798999  | Personal     | 
+
+Scenario: Add single user and edit its data
+    Given the following contacts have been created 
+        |name     |email               |phone       |contact-type  |
+        |deekshya |deekshyap@gmail.com |9567989999  | personal     |
+
```
Comment (kiranadh1452) [inline Features4/tests/feature/editContact.feature]:
```suggestion
```

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

File: Features4/tests/feature/editContact.feature (hunk 3 above)
Comment (kiranadh1452) [reply Features4/tests/feature/editContact.feature]: @Dikshyapoudel12 please resolve only after addressing this.

**Review by kiranadh1452 — APPROVED** [review]

**Pull request merged**